inference/core/workflows/core_steps/utils/dominant_color/v1.py

```diff
@@ -0,0 +1,79 @@
+from typing import List, Literal, Type, Union
+
+import numpy as np
+from pydantic import AliasChoices, ConfigDict, Field
+from sklearn.cluster import MiniBatchKMeans
+
+from inference.core.workflows.execution_engine.entities.base import (
+    OutputDefinition,
+    WorkflowImageData,
+)
+from inference.core.workflows.execution_engine.entities.types import (
+    BATCH_OF_INTEGER_KIND,
+    StepOutputImageSelector,
+    WorkflowImageSelector,
+)
+from inference.core.workflows.prototypes.block import (
+    BlockResult,
+    WorkflowBlock,
+    WorkflowBlockManifest,
+)
+
+TYPE: str = "roboflow_core/dominant_color@v1"
+SHORT_DESCRIPTION: str = "Get the dominant color of an image."
+LONG_DESCRIPTION: str = "Get the dominant color of an image as a list of RGB values."
+
+
+class DominantColorManifest(WorkflowBlockManifest):
+    type: Literal[f"{TYPE}"]
+    model_config = ConfigDict(
+        json_schema_extra={
+            "name": "Dominant Color",
+            "version": "v1",
+            "short_description": SHORT_DESCRIPTION,
+            "long_description": LONG_DESCRIPTION,
+            "license": "Apache-2.0",
+            "block_type": "util",
+        }
+    )
+
+    image: Union[WorkflowImageSelector, StepOutputImageSelector] = Field(
+        title="Input Image",
+        description="The input image for this step.",
+        examples=["$inputs.image", "$steps.cropping.crops"],
+        validation_alias=AliasChoices("image", "images"),
+    )
+
+    @classmethod
+    def describe_outputs(cls) -> List[OutputDefinition]:
+        return [
+            OutputDefinition(name="rgb_color", kind=[BATCH_OF_INTEGER_KIND]),
```

Comment:
let's change into list of values kind for now

Reply:
complete ✅

Should I remove the BATCH_OF_INTERGER_KIND type in `inference/core/workflows/execution_engine/entities/types.py`?